agent/logger/log_windows_test.go

```diff
@@ -341,47 +341,109 @@ func TestSeelogConfig_FileLevelDefault(t *testing.T) {
 }
 
 func TestSetLevel(t *testing.T) {
-	os.Setenv(LOGLEVEL_ON_INSTANCE_ENV_VAR, "crit")
-	os.Setenv(LOGLEVEL_ENV_VAR, "debug")
-
 	resetEnv := func() {
 		os.Unsetenv(LOGLEVEL_ENV_VAR)
 		os.Unsetenv(LOGLEVEL_ON_INSTANCE_ENV_VAR)
+		os.Unsetenv(LOG_DRIVER_ENV_VAR)
 	}
-	defer resetEnv()
+	resetEnv()
 
-	Config = &logConfig{
-		logfile:       "foo.log",
-		driverLevel:   DEFAULT_LOGLEVEL,
-		instanceLevel: setInstanceLevelDefault(),
-		RolloverType:  DEFAULT_ROLLOVER_TYPE,
-		outputFormat:  DEFAULT_OUTPUT_FORMAT,
-		MaxFileSizeMB: DEFAULT_MAX_FILE_SIZE,
-		MaxRollCount:  DEFAULT_MAX_ROLL_COUNT,
+	testcases := []struct {
```

Comment:
is there any difference between this test and the one in log_test.go? if not, ideally we can rename `log_test.go` to `log_unix_test.go` (since it has build constraint `!windows`), and create another log_test.go that runs on both unix and windows, and put TestSetLevel in that file